flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/ListView.java

```diff
@@ -57,15 +60,15 @@
  *    public ListView<String> list = new ListView<>();
  *
  *    // or explicit:
- *    // {@literal @}DataTypeHint("ARRAY<STRING>")
+ *    // @DataTypeHint("ARRAY < STRING >")
  *    // public ListView<String> list = new ListView<>();
```

Comment:
looks like  `// public ListView<String> list = new ListView<>();` should be removed

Reply:
no, this is intentional. otherwise two lists would exist in the accumulator